specification/communication/data-plane/CallAutomation/preview/2024-11-15-preview/communicationservicescallautomation.json

```diff
@@ -2230,6 +2230,107 @@
           "additionalProperties": {
             "type": "string"
           }
+        },
+        "callDetails": {
```

Comment:
add a polymorphic set of properties here; add a 'kind' for callDetails, one for Teams AA, another as a generic key/value pair, etc...